app/controllers/houses_controller.rb

```diff
@@ -1,60 +1,113 @@
 class HousesController < ApplicationController
-
   def index
-    @houses = order_list(House.eager_all, :id)
-
-    format({ serialized_data: @houses })
+    @houses = Parliament::Request.new.houses.get
   end
 
   def show
-    @house = House.eager_find(params[:id]) or not_found
-
-    format({ serialized_data: @house })
+    house_id = params[:id]
+    data = Parliament::Request.new.houses(house_id).get
+    @house = data.filter('http://id.ukpds.org/schema/House').first
   end
 
   def members
-    @house = House.eager_find(params[:house_id], 'members') or not_found
-    @members = order_list(@house.members, :surname, :forename)
-
-    format({ serialized_data: @house })
+    house_id = params[:house_id]
+    data = Parliament::Request.new.houses(house_id).members.get
+    @house = data.filter('http://id.ukpds.org/schema/House').first
```

Comment:
Fixed all these issues (and below comments) in both people_controller and houses_controller.